actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -88,13 +92,15 @@
 
     private CompletableFuture<Void> hostingActive = new Task<>();
 
-    private int maxLocalAddressCacheCount = 10_000;
-
     private NodeSelectorExtension nodeSelector;
 
-    public Hosting()
+    public Hosting(final int maxLocalAddressCacheCount, final long defaultActorTTL)
```

Comment:
Yep. I'm wondering if @ceckhardt is right and we should just be using the access time and not have a maximum.
It's very difficult to guess how many actors a frontend might touch, and the entries in this cache are small so it seems unlikely there are any major memory concerns.